test/qa/lib/__init__.py

```diff
@@ -204,6 +201,13 @@ def set_threat_protection_lite(dns):
         print("WARNING:", ex)
 
 
+def set_dns(dns):
```

Comment:
What is the purpose of this? Seems like it just allows ignoring the error, which doesn't seem like a good thing.

Reply:
Ah, I see there's already a comment above.